### DISCUSSION TITLE
.net 8 core web app  Can code server build

### DISCUSSION THREAD

**Discussion by gunduzcagri** [coder/code-server#6952] [General]
Hello, I tried some plugins but I could not build ".net 8 core web app". How can I do it?

**Comment by code-asher** [comment]
Are you having a problem with a specific plugin?  If so, which is it?  What errors are you seeing?

**Reply by gunduzcagri** [reply]
![Ekran Görüntüsü - 2024-08-21 15-28-14](https://github.com/user-attachments/assets/25716291-c5cd-498f-87b6-a57b7b0f590c)
There is no slotion screen  on code server  but i have on vs code 
![WhatsApp Image 2024-08-21 at 15 26 19](https://github.com/user-attachments/assets/432e1eee-024f-4d54-b278-771607ef352d)

I cant find some microsoft plugins  on code server plugins market

**Reply by code-asher** [reply]
Ah that is provided by the C# extension, right?

Microsoft does not allow code-server to use their marketplace, so we have to use [Open VSX](https://open-vsx.org/) instead.  It looks like Microsoft has not made their C# extension available on Open VSX and it looks like they have no intention to do this: https://github.com/dotnet/vscode-csharp/issues/3853

So, the only way to use it would be to download the `.vsix` file and install it into code-server manually, unfortunately.

**Reply by code-asher** [reply]
It might still not work even if you manually install though, because I think parts of it are only allowed to be ran in Microsoft's build of VS Code.

Basically, the problem is licensing
